### PR TITLE
SRVCOM-2431: transform xrefs to serverless content into links

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.10+

Issue:
[SRVCOM-2431](https://issues.redhat.com//browse/SRVCOM-2431)

Link to docs preview:

https://60450--docspreview.netlify.app/openshift-enterprise/latest/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#prerequisites_odc-creating-applications-using-developer-perspective

https://60450--docspreview.netlify.app/openshift-enterprise/latest/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#additional-resources_odc-creating-applications-using-developer-perspective

https://60450--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/kn-cli-tools.html#kn-cli-tools-installing-kn

https://60450--docspreview.netlify.app/openshift-enterprise/latest/security/container_security/security-build.html#additional-resources_security-build

https://60450--docspreview.netlify.app/openshift-enterprise/latest/hardware_enablement/kmm-kernel-module-management.html#kmm-must-gather-tool_kernel-module-management-operator

### CODE REVIEW

File: applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ The Managed services option is also available if the RHOAS Operator is installed
...
+Note that certain options, such as *Pipelines*, *Event Source*, and *Import Virtual Machines*, are displayed only when the xref:../../cicd/pipelines/installing-pipelines.adoc#op-installing-pipelines-operator-in-web-console_installing-pipelines[OpenShift Pipelines Operator], link:https://docs.openshift.com/serverless/1.28//install/install-serverless-operator.html#serverless-install-web-console_install-serverless-operator[{ServerlessOperatorName}], and xref:../../virt/install/installing-virt-web.adoc#virt-subscribing-to-the-catalog_installing-virt-web[OpenShift Virtualization Operator] are installed, respectively.
```
Comment (abrennan89) [inline applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc:53]:
```suggestion
Note that certain options, such as *Pipelines*, *Event Source*, and *Import Virtual Machines*, are displayed only when the xref:../../cicd/pipelines/installing-pipelines.adoc#op-installing-pipelines-operator-in-web-console_installing-pipelines[OpenShift Pipelines Operator], link:https://docs.openshift.com/serverless/1.28/install/install-serverless-operator.html#serverless-install-web-console_install-serverless-operator[{ServerlessOperatorName}], and xref:../../virt/install/installing-virt-web.adoc#virt-subscribing-to-the-catalog_installing-virt-web[OpenShift Virtualization Operator] are installed, respectively.
```

Comment (abrennan89) [reply applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc:53]: looks like there's an extra `/` here, but the link works fine in the preview 🤷🏻

File: security/container_security/security-build.adoc [hunk 2]
```diff
@@ -35,6 +35,6 @@ include::modules/security-build-knative.adoc[leveloffset=+1]
 * xref:../../cicd/builds/creating-build-inputs.adoc#builds-input-secrets-configmaps_creating-build-inputs[Input secrets and config maps]
 ifndef::openshift-origin[]
 // **This section is being moved, update the link once content is in.* add link here[The CI/CD methodology and practice]
-* xref:../../serverless/about/about-serverless.adoc#about-serverless[About OpenShift Serverless]
+* link:https://docs.openshift.com/serverless/1.28/about/about-serverless.html[About OpenShift Serverless]
```
Comment (abrennan89) [inline security/container_security/security-build.adoc:38]: I'd rename to "OpenShift Serverless overview" since that's the name of the page

**Review by abrennan89 — COMMENTED** [review]
A couple of small comments, otherwise lgtm

File: applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc (hunk 1 above)
Comment (gabriel-rh) [reply applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc]: Fixed

**Pull request merged**